mlflow/models/evaluation/base.py

```diff
@@ -1539,11 +1553,17 @@ def pred_sample(eval_df, _builtin_metrics, _artifacts_dir):
     ) = _normalize_evaluators_and_evaluator_config_args(evaluators, evaluator_config)
 
     with _start_run_or_reuse_active_run() as run_id:
+        # Convert data to `mlflow.data.dataset.Dataset`.
+        data = _convert_data_to_mlflow_dataset(
+            data=data,
+            targets=targets,
+            dataset_path=dataset_path,
```

Comment:
`dataset_path` is an optional arg. what do we want to do if this is not specified? the `CodeDatasetSource` probably makes sense here.

Reply:
Done!